flight/Modules/Autotune/autotune.c

```diff
@@ -191,6 +193,8 @@ static void AutotuneTask(void *parameters)
 		const uint32_t PREPARE_TIME = 2000;
 		const uint32_t MEASURE_TIME = 60000;
 
+		static uint32_t updateCounter = 0;
```

Comment:
We should decide definitively what to do for PRs touching existing files.

1. Match existing convention in file.
2. Use a different style for new variables.
3. Re-style the whole file.

3 makes review difficult.  I'm willing to do 1 or 2-- they're each ugly in different ways.